programmers/난이도별/level01.크레인_인형뽑기_게임/yoojinhee03.java

```diff
@@ -0,0 +1,36 @@
+import java.util.*;
+
+public class yoojinhee03 {
+
+	public static void main(String[] args) {
+		
+		int [][] board= {{0,0,0,0,0},{0,0,1,0,3},{0,2,5,0,1},{4,2,4,4,2},{3,5,1,3,1}};
+		int []moves= {1,5,3,5,1,2,1,4};
+		
+		System.out.println(solution(board,moves));
+		
+	}
+	public static int solution(int[][] board, int[] moves) {
+        
+        int answer = 0;
+        ArrayList array = new ArrayList();
```

Comment:
```suggestion
        List<Integer> list = new ArrayList();
```
- List는 ArrayList의 interface 입니다.
- 반대로 ArrayList는 List의 implement 입니다.
그래서 변수명은 보편적으로 `list`에 초점을 맞추는게 좋을 것 같아요!